docs/src/modules/java/pages/kickstart.adoc

```diff
@@ -165,6 +165,55 @@ The Maven failsafe plugin runs the integration tests when the `it` profile is en
 mvn verify -Pit
 ----
 
+=== Unit Tests for Event Sourced Entities with autogenerated Testkit
+
+We can test an Event Sourced Entity implementation with the AkkaserverlessTestkit. This class is autogenerated by Akkaserverless when a project with an EventSourceEntity is compiled. Creating an ad-hoc TestKit. 
```

Comment:
I don't think we write Akkaserverless in one word like that. Akka Serverless if it's in text and not referring to a specific class.

Reply:
`EventSourcedEntity` or Event Sourced Entity

Reply:
done